Dockerfile.arm32v7

```diff
@@ -42,6 +42,9 @@ RUN grep -v 'tensorflow' requirements.txt | python -m pip install -r /dev/stdin
 
 COPY . /workspace
 
+# Template substitution: Replace @model@ with the proper model name
```

Comment:
At first glance it seems like the user is supposed to manually change `@model@` to the actual model name. Maybe this is better?

```suggestion
# Template substitution: Replaces @model@ with the proper model name
```